src/scripts/main.js

```diff
@@ -1 +1,65 @@
 'use strict';
+
+const body = document.querySelector('body');
+
+function succesMessage(succesText) {
+  const newMessage = document.createElement('div');
+
+  newMessage.classList.add('succes');
+  newMessage.dataset.qa = 'notification';
+  newMessage.innerText = succesText;
+  body.append(newMessage);
+}
+
+function errorMessage(errorText) {
+  const newMessage = document.createElement('div');
+
+  newMessage.classList.add('warning');
+  newMessage.dataset.qa = 'notification';
+  newMessage.innerText = errorText;
+  body.append(newMessage);
+}
```

Comment:
actually, you can combine it in one function, and just pass the class as the second argument